src/ucp/wireup/wireup_cm.c

```diff
@@ -942,8 +942,7 @@ void ucp_cm_server_conn_request_cb(uct_listener_h listener, void *arg,
 err_reject:
     status = uct_listener_reject(listener, conn_request);
     if (status != UCS_OK) {
-        ucs_warn("failed to reject connect request %p on listener %p",
-                 conn_request, listener);
+        ucs_warn("failed to reject connect request on listener %p", listener);
```

Comment:
can we add Coverity exception here? since ucs_warn does really not dereference the pointer, just prints it

Reply:
you mean to add a comment to make coverity ignore this?

Reply:
yes